api/datasources.production.js

```diff
@@ -1,12 +0,0 @@
-module.exports = {
```

Comment:
Ohh, hadn't seen how that was handled in datasources.local.js. Totally right. Closing this PR.